### ISSUE TITLE
[import/newline-after-import] can't Identify require statement when module name is declared by template string

### ISSUE THREAD

**Issue by tjx666** [benmosher/eslint-plugin-import#1614]
test code:

```javascript
const assert = require('assert');
const generate = require(`../../src/0118-Pascal's Triangle/generate`);

describe(`#0118-Pascal's Triangle`, () => {});
```

report error:

```
Expected 1 empty line after require statement not followed by another require.eslint(import/newline-after-import)
```
![screenshot_2020-01-17_17-11-11](https://user-images.githubusercontent.com/41773861/72599474-baa69b00-394c-11ea-9ba2-d323a7865fe5.png)

**Comment by ljharb** [comment]
It seems reasonable to support detecting a template literal as long as it has no substitutions.
